pkg/regionpicker/regionpicker.go

```diff
@@ -0,0 +1,91 @@
+package regionpicker
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// clouds is the global map of all supported clouds
+var clouds = map[CloudName]Cloud{
+	CloudGCP: &GCP{},
+}
+
+// getPing hits an endpoint and returns the ping time
+func getPing(url string) (time.Duration, error) {
```

Comment:
This should really be a method on `region` IMO -- it would also read better in `best` -- `r.Duration(ctx)` ...

Reply:
Yeah, fair. Will fix